kannon/master.py

```diff
@@ -28,9 +27,10 @@ def __init__(
         # kannon resources
         job_prefix: str,
         path_child_script: str = "./run_child.py",
-        env_to_inherit: Optional[List[str]] = None,
-        master_pod_name: Optional[str] = None,
-        master_pod_uid: Optional[str] = None,
+        env_to_inherit: list[str] | None = None,
+        master_pod_name: str | None = None,
+        master_pod_uid: str | None = None,
+        dynamic_config_path: str | None = None,
```

Comment:
Changed type annotation styles to more newer one without being dependent on typing